ttt.py

```diff
@@ -92,7 +91,16 @@ def transcribe_fasterwhisper(calljson, audiofile):
 
     # This whisper wants the path, not bytes but we need to cast it from pathlib to str
     audiofile = str(audiofile)
-    segments, info = model.transcribe(audiofile, beam_size=5, vad_filter=vad_filter)
+    # We are going to set the vad parameters to half a second although env variable still turns
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_refinement):** Consider making VAD parameters configurable.

Hardcoding the `min_silence_duration_ms` to 500 might not be flexible for all use cases. Consider making this configurable through an environment variable.